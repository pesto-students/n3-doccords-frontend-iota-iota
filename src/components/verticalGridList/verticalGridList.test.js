import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import VerticalGridList from "components/verticalGridList";

describe("VerticalGridList", () => {
  const list = [
    {
      healthTopicId: "sZX6do6jozYrHnYfBVeH",
      title: "Aging 2",
      picture:
        "https://firebasestorage.googleapis.com/v0/b/doccords-55659.appspot.com/o/1625396784996-Aging.jpeg_1625396785947?alt=media",
      createdAt: "2021-07-04T11:06:38.485Z",
    },
  ];

  it("rendering without Error", () => {
    const wrapper = shallow(
      <VerticalGridList healthTopics={list} handleClick={() => {}} />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
