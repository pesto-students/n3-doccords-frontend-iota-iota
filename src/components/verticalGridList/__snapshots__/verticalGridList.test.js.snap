// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerticalGridList rendering without Error 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(GridList))
    cellHeight={180}
    className="makeStyles-gridList-2"
    cols={3}
  >
    <WithStyles(ForwardRef(GridListTile))
      key="sZX6do6jozYrHnYfBVeH"
      onClick={[Function]}
    >
      <img
        alt="Aging 2"
        src="https://firebasestorage.googleapis.com/v0/b/doccords-55659.appspot.com/o/1625396784996-Aging.jpeg_1625396785947?alt=media"
      />
      <WithStyles(ForwardRef(GridListTileBar))
        title="Aging 2"
      />
    </WithStyles(ForwardRef(GridListTile))>
  </WithStyles(ForwardRef(GridList))>
</div>
`;
